.vr_settings.yaml

```diff
@@ -1,18 +1,14 @@
 --- !ruby/object:VR_ENV
-width: 800
-height: 600
+width: 1113
+height: 666
 panel_pos: 360
 run_command_line: ruby tictactoe
 open_files:
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/README.md"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/player.rb"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/Board.rb"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/tictactoe.rb"
-- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/tictactoe"
+- "/home/eric/visualruby/tictactoe/tictactoe.gemspec"
```

Comment:
Yea, I'll figure out the .gitignore way to do this.  This is a settings file generated by vr.  It doesn't need to be uploaded each time.  
E 

      From: Randall Reed <notifications@github.com>
 To: randallreedjr/tictactoe_vr <tictactoe_vr@noreply.github.com> 
Cc: Eric Cunningham <Beagle4321_2000@yahoo.com>; Author <author@noreply.github.com>
 Sent: Saturday, May 21, 2016 1:30 PM
 Subject: Re: [randallreedjr/tictactoe_vr] fixed command line executable. (#2)
   
In .vr_settings.yaml:>  panel_pos: 360
>  run_command_line: ruby tictactoe
>  open_files:
> -- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/README.md"
> -- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/player.rb"
> -- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/Board.rb"
> -- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/bin/tictactoe.rb"
> -- "/Users/rcrjr/Flatiron/Prework/tictactoe_vr/tictactoe"
> +- "/home/eric/visualruby/tictactoe/tictactoe.gemspec"
I think this path needs to be made more generic, or this file excluded from the project (added to .gitignore).—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub